intera_examples/scripts/go_to_joint_angles_in_contact.py

```diff
@@ -146,6 +146,13 @@ def main():
         "-kn", "--K_nullspace", type=float,
         nargs='+', default=[5.0, 10.0, 5.0, 10.0, 5.0, 10.0, 5.0],
         help="A list of desired nullspace stiffnesses, one for each of the 7 joints (a single value can be provided to apply the same value to all the directions) -- units are in (Nm/rad)")
+    parser.add_argument(
+        "-ddifc",  "--disable_damping_in_force_control", action='store_true', default=False,
```

Comment:
Yeah, I agree that these are getting long. When we do have a lot of options though, which is often the case, these short-hands do help. We can take another iteration in the future to fix all of them at once.